app/store/chatList.go

```diff
@@ -300,6 +302,51 @@ func (s *Sessions) CreateSessionForE164(tel string, UUID string) *Session {
 	return ses
 }
 
+func (s *Sessions) CreateSessionForUUID(UUID string) *Session {
+	contact := GetContactForUUID(UUID)
+	ses := &Session{
+		Tel:          contact.Tel,
+		Name:         contact.Name,
+		Active:       true,
+		IsGroup:      false,
+		Notification: true,
+		UUID:         UUID,
+	}
+	if s.Len == 0 {
+		ses.ID = 1
+	}
+	ses, err := SaveSession(ses)
+	log.Errorln("[axolotl] CreateSessionForUUID failed:", ses.ID)
```

Comment:
Why an error log here? That's the one I had during testing, but that looks like a mistake.

Reply:
That's right it's not necessary